edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -110,6 +110,15 @@ def _return_sku(self, seats):
         """ Return dict of sku for a course run. """
         return {seat['type']: seat['sku'] for seat in seats if seat['type'] in REQUIRED_SEATS_TYPES}
 
+    def build_course_owners(self, course):
+        owners_length = len(course.get('owners'))
+        counter = 0
+        owners = ''
+        while counter < owners_length:
```

Comment:
I believe you can do something like this, which will be more cleaner.
```
owners = []
for owner in course.get('owners'):
    owners.append(owner['key'].replace('_', ' '))
 return ' '.join(owners)
```

Reply:
I agree with @tasawernawaz . I guess one should avoid using while/counter construct and use iterators